### PR TITLE
Add Icons

### PR DESCRIPTION
**Pull request by Lorenzo-Mariano**

### CODE REVIEW

File: public/index.html [hunk 1]
```diff
@@ -1,12 +1,52 @@
...
+                    Philippines (aw hell naw dawg)
+                </div>
+
+                <div class="city">
+                    Marawi city
+                </div>
+            </div>
+        </div>
+
+        <div class="spacer-3"></div>
```
Comment (mikolertesx) [inline public/index.html]: Why not just spacer, instead of `spacer-1`, `spacer-2`, `spacer-3`.

File: public/styles.css [hunk 2]
```diff
@@ -0,0 +1,95 @@
...
+  font-family: "Poppins", sans-serif;
+  font-weight: 300;
+}
+
+/* def gonna wanna change this, i just forgor how */
+/* thats how much i have deteriorated */
+/* also commenting as much as i want cuz idk */
+.spacer-1 {
+  height: 42px;
+}
```
Comment (mikolertesx) [inline public/styles.css]: Same comment as before, why `spacer-1`, [...], instaed of `spacer`

File: src/index.js [hunk 3]
```diff
@@ -1,12 +1,33 @@
...
+}
+else if (userTime >= 16 && userTime <= 18) {
+    console.log('This is sunset')
+    document.body.style.backgroundColor = '#FE8714'
+}
+else if (userTime >= 19 && userTime <= 4) {
+    console.log('its nighttime')
+    document.body.style.backgroundColor = '#27445C'
+}
+
```
Comment (mikolertesx) [inline src/index.js:32]: This can stay like this for now, we should later move this to a different file.

**Review by mikolertesx — CHANGES_REQUESTED** [review]
Please change this.

File: src/index.js (hunk 3 above)
Comment (Lorenzo-Mariano) [reply src/index.js:32]: Understood.

**Review by mikolertesx — APPROVED** [review]
LGTM

**Pull request merged**